CONTRIBUTING.md

```diff
@@ -247,9 +266,7 @@ As an example, let's assume you have created the svgs for Redhat and Amazon Web
   </li>
   <li>Create a separate pull request (PR) for each Icon.
     <ul>
-      <li>This means you would have to create two PRs</li>
-      <li>For Amazon Web Services, the branch name would be icons/amazonwebservices. </li>
-      <li>For Redhat, the branch name would be icons/redhat. </li>
+      <li>This means you would have to create two PRs for each Icon.</li>
```

Comment:
Nope. You don't have to create two pull requests for each icon. I think the version before is right. Or am I missing something? :smile:

Reply:
For this, I was referring to two different sets of icons. In the examples, I was submitting AWS and React to demonstrate when people want to tackle multiple technology. I think I'll rephrase it so it's clearer